tests/controllers/authenticationRoute/userRegister.test.js

```diff
@@ -0,0 +1,23 @@
+const request = require("supertest");
+const { server } = require("../../../src/server/server");
+
+describe("Testing userLogin endpoint", function () {
+  test("POST /authentication/login should return 400 if bad request body", async function () {
+    const response = await request(server)
+      .post("/authentication/login")
+      .send({});
+    expect(response.status).toBe(400);
+  });
+
+  /*
+   * Nedan test funkar inte än, måste kolla upp kopplingen till databasen varför den delen inte funkar
+   */
+  /*
+  test("POST /authentication/login should return 200 if good login", async function () {
+    const response = await request(server)
+      .post("/authentication/login")
+      .send({ username: "Mika", password: "Choklad" });
```

Comment:
Ska vi verkligen ha ditt username och psaaword postat på Github?

Reply:
Uppdaterat detta till ett test konto istället

Reply:
Borde vi inte ta bort den registrerade testanvändaren efter att ha kört test-sviten istället? Annars blir det ju samma sak som att ha ditt lösenord upplagt, om vi inte lagt in någon typ av begränsning på test-kontot och vilka åtgärder det får göra i databasen?

Reply:
Kan lägga till att ta bort testkontot men behöver få min andra pull request godkänt för att kunna ta bort användaren för den endpointen är inte tillgänglig i denna pull-request än

Reply:
Men pulla in den i den här requesten då? Den behöver den ju inte ha blivit godkänd till main för det? Oavsett så har den andra requesten gått igenom nu ser det ut som?